changelog/Microsoft.DirectoryServices.json

```
@@ -9873,6 +9873,68 @@
       "CreatedDateTime": "2021-11-15T20:42:47.116895Z",
       "WorkloadArea": "Identity and access",
       "SubArea": "Identity and sign-in"
+    },
+    {
+      "changelog": [
+        {
+          "ChangeList": [
+            {
+              "Id": "34b0a990-a3c9-4b70-92ac-a8ac93bf4860",
+              "ApiChange": "EnumType",
+              "ChangedApiName": "x509CertificateAuthenticationMode",
+              "ChangeType": "Addition",
+              "Description": "Added the [x509CertificateAuthenticationMode](https://docs.microsoft.com/en-us/?view=graph-rest-beta) resource",
+              "Target": "x509CertificateAuthenticationMode"
+            },
+            {
+              "Id": "34b0a990-a3c9-4b70-92ac-a8ac93bf4860",
+              "ApiChange": "EnumType",
+              "ChangedApiName": "x509CertificateRuleType",
+              "ChangeType": "Addition",
+              "Description": "Added the [x509CertificateRuleType](https://docs.microsoft.com/en-us/?view=graph-rest-beta) resource",
+              "Target": "x509CertificateRuleType"
+            },
+            {
+              "Id": "34b0a990-a3c9-4b70-92ac-a8ac93bf4860",
+              "ApiChange": "Resource",
+              "ChangedApiName": "x509CertificateAuthenticationModeConfiguration",
+              "ChangeType": "Addition",
+              "Description": "Added the [x509CertificateAuthenticationModeConfiguration](https://docs.microsoft.com/en-us/graph/api/resources/x509CertificateAuthenticationModeConfiguration?view=graph-rest-beta) resource",
```

Suggestions to enhance the changelog description and provide a little more context
```suggestion
              "Description": "Added the [x509CertificateAuthenticationModeConfiguration](https://docs.microsoft.com/en-us/graph/api/resources/x509CertificateAuthenticationModeConfiguration?view=graph-rest-beta) resource. Represents the details of the Azure AD native Certificate-Based Authentication (CBA) in the tenant including whether the authentication method is enabled or disabled and the users and groups who can register and use it.",
```

---

fixed